controllers/model/prometheus_resources.go

```diff
@@ -470,3 +472,11 @@ func GetPrometheusOperatorResourceRequirement(cr *v1.Observability) v13.Resource
 	}
 	return v13.ResourceRequirements{}
 }
+func GetPrometheusStorageSize(cr *v1.Observability, indexes []v1.RepositoryIndex) string {
+	customPrometheusStorageSize := ""
```

Comment:
shouldn't this also default to `250Gi`?

Reply:
You're right @pb82. I should update this after including `250Gi` as default in resources. Thanks!